pkg/networkserver/downlink_internal_test.go

```diff
@@ -205,7 +205,7 @@ func TestGenerateDataDownlink(t *testing.T) {
 				},
 				LorawanPhyVersion:       ttnpb.RP001_V1_1_REV_B,
 				FrequencyPlanId:         band.EU_863_870,
-				LastDevStatusReceivedAt: TimePtr(time.Unix(42, 0)),
+				LastDevStatusReceivedAt: ttnpb.ProtoTime(TimePtr(time.Unix(42, 0))),
```

Comment:
`ttnpb.ProtoTimePtr(x)` instead of `ttnpb.ProtoTime(TimePtr(x))`. You can also check if there are still uses for `TimePtr` after you do these changes - if not, feel free to remove it.